### PR TITLE
fix: support for request options in table data source

### PR DESCRIPTION
**Pull request by arjunlalb**
## Description
support for request options in table data source

### CODE REVIEW

File: projects/distributed-tracing/src/shared/dashboard/data/graphql/table/table-data-source.model.ts [hunk 1]
```diff
@@ -20,13 +20,17 @@ export abstract class TableDataSourceModel extends GraphQlDataSourceModel<TableD
...
       getData: request =>
-        this.query(filters => this.buildGraphQlRequest(filters, request)).pipe(
+        this.query(filters => this.buildGraphQlRequest(filters, request), this.buildGraphqlRequestOptions()).pipe(
           map(response => this.buildTableResponse(response, request))
         ),
       getScope: () => this.getScope()
     });
   }
 
+  public buildGraphqlRequestOptions(): GraphQlRequestOptions {
```
Comment (aaron-steinfeld) [inline projects/distributed-tracing/src/shared/dashboard/data/graphql/table/table-data-source.model.ts:30]:
```suggestion
  protected buildGraphqlRequestOptions(): GraphQlRequestOptions {
```

**Review by aaron-steinfeld — APPROVED** [review]

**Review by aaron-steinfeld — APPROVED** [review]

File: projects/observability/src/pages/explorer/explorer.component.test.ts [hunk 2]
```diff
@@ -141,7 +141,7 @@ describe('Explorer component', () => {
         limit: 500,
         interval: new TimeDuration(15, TimeUnit.Second)
       }),
-      undefined
+      expect.objectContaining({})
```
Comment (aaron-steinfeld) [inline projects/observability/src/pages/explorer/explorer.component.test.ts:144]: nit - this matches any object - we could also just put `{}` here

Comment (arjunlalb) [reply projects/observability/src/pages/explorer/explorer.component.test.ts:144]: weirdly enough, that doesn't work. I couldn't figure out why. Any ideas?

Comment (aaron-steinfeld) [reply projects/observability/src/pages/explorer/explorer.component.test.ts:144]: What was the error? It should say what it expected/got

**Pull request merged**